### ISSUE TITLE
Combat and stunning

### ISSUE THREAD

**Issue by UndeadNappist** [BeeStation/NSV13#79]
People have talked about how combat is unbalanced (which it is) so here is some thought and discussion on various key issues and things various servers have used.

**Gameflow:**
What are you going for? Do you want a combat system that's perfectly balanced, relies on hp, and therefore strip stuns out of the game? Do you want a more roleplay-heavy style where it's hard to kill anyone for any reason? What do you want, what do you want players to do, and how do you go about encouraging that?
The basic questions you should ask yourself before rewriting combat - and understanding why things are what they are.

**Stunning and capturing:**
Stunning and capturing need to be relatively easy _in comparison to_ murder. This is to keep new and old players from simply switching to killing everyone. A security guard will not taser people if it takes multiple taser shots and then they have to wrestle a target to the ground with handcuffs, no matter how realistic it is, when they can shoot them once in the torso with a shotgun.

**Realism:**
Needs to take a back seat to game-ism. It's useful as an information resource to make the game more interesting and deep, but not as a punishment mechanic.
See: The whole lung-popping business of bay and actually researching barotrauama and lung rupturing.

**TG/Goon-Combat**
I'm putting them in the same category because the only difference is you can cry for help in Goon even when incapacitated basically instantly. Very stun heavy, very high lethality, fast-paced, anything short of nearly killing you won't put you out for very long. You can whip out any bullshit tool instantly and use it.
**Bay**
Nearly no stuns, or the stuns that exist are basically resisted for arbitrary reasons to the point people would rather beat each other to death. Medical system that results in you rapidly dying from any injury. Has a lot of RNG built into gun accuracy and in straight combat gear-checks you.
Still has the same insta-stunning as goon/tg if you know what you're doing.
Basically the worst of all worlds, although you can see their vestigial attempts to promote squad tactics and slow down combat.
**CM/CM**
What Bay was trying to do but failed and taken in a different direction. No stuns, crippling injuries can happen but don't stop you from fighting until you drop, weapon damage is fairly balanced across the board, high-lethality for the expectation that you'll just respawn.
**Lifeweb**
Focus on a unique combat system that concentrates on a combination of paying attention and using the tools available and stat-checking RNG based on what role you've chosen. Discourages non-coms to perform combat, danger of even minor injuries developing infection discourages random fights, respawn is determined entirely by whoever kills you. Unless you both die, then it's time for zombies.
Adds a delay between drawing a weapon and attacking, doesn't let you access your backpack while you're wearing it. Looking at someone can determine combat readiness. Lack of skills in certain directions means keeping specific roles alive because you need their help to survive long-term.
Has some dumb RNG permadeath stuff, is dark and grainy like a decaying VCR tape to the point it's difficult to see anything going on.

tl;dr
Combat isn't just about 'combat.' It's about other pressures that might encourage or discourage people from killing each other, both social and physical, coupled with access to equipment and post/pre-consequences of performing combat.
Fundamentally, ss13 has always been a high lethality game. There's no surviving being sucked out into deep space without gear.
